### PR TITLE
add events pages and graphics for hacktoberfest

### PR DESCRIPTION
**Pull request by davidwyld**
Feedback/edits appreciated, btw I'll add copies of the second event for the third and fourth if everyone's happy.

image

### CODE REVIEW

File: _events/hacktoberfest-2020-10-06.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Many of the world's most influential software projects are open source - and it doesn't take a master developer to contribute meaningful code and help them improve. Hacktoberfest is all about encouraging meaningful contributions to the open-source ecosystem
+and Hacktoberfest meetups are **for beginners and veterans alike**!
+
+[Research Software Engineering (RSE) Sheffield](/),
+are organising a **series in of hackathons** during October
+as part of [**Hacktoberfest 2020**][hacktoberfest],
+a **global initiative to encourage participation in open-source software projects**.
+
+The focus of this initial session will be on using GitHub to explore and contribute to open-source projects. We'll start with an introduction to how to contribute via git and GitHub, then move on to some hands-on tutorials using either RStudio, GitKraken/VScode or command line git (your choice!).
```
Comment (bobturneruk) [inline _events/hacktoberfest-2020-10-06.md]: I'd say just GitKraken not GitKraken/VScode, unless we're sure we can get another person to deliver as well.

**Review by bobturneruk — APPROVED** [review]
Looking good.

File: _events/hacktoberfest-2020-10-06.md (hunk 1 above)
Comment (davidwyld) [reply _events/hacktoberfest-2020-10-06.md]: Cool, I wasn't sure which one you'd decided to go with.

Comment (bobturneruk) [reply _events/hacktoberfest-2020-10-06.md]: I think Kraken as we've definitely got materials for that.

**Comment by willfurnass** [comment]
@davidwilby  Looks good.  Any thoughts on a show and tell at the last session?  Also, worth expanding a little more on why folks might want to get involved?  (meet people at TUOS with interest in coding but different skill levels; learn/share ways of using git to contribute to projects etc)  Also, want to include anything about the RSE team helping people find interesting software to work on via a curated set of projects/Issues?

**Comment by willfurnass** [comment]
@davidwilby  Looks good.  Any thoughts on a show and tell at the last session?  Also, worth expanding a little more on why folks might want to get involved?  (meet people at TUOS with interest in coding but different skill levels; learn/share ways of using git to contribute to projects etc; could also link to #309 )  Also, want to include anything about the RSE team helping people find interesting software to work on via a curated set of projects/Issues?

**Comment by bobturneruk** [comment]
In light of @willfurnass comments - maybe best to get a version up on the site, then refine?

**Comment by davidwyld** [comment]
> @davidwilby Looks good. Any thoughts on a show and tell at the last session? Also, worth expanding a little more on why folks might want to get involved? (meet people at TUOS with interest in coding but different skill levels; learn/share ways of using git to contribute to projects etc; could also link to #309 ) Also, want to include anything about the RSE team helping people find interesting software to work on via a curated set of projects/Issues?

Thanks @willfurnass I've fleshed out the details as per your suggestions, I like the idea of a showcase at the end of the month, will sort that out in due course.

I'll get this up on the site and we can tweak more afterwards.

**Pull request merged**